src/__tests__/hydrateSatelliteObject.test.js

```diff
@@ -188,6 +189,45 @@ describe('hydrateSatelliteObject', function () {
     expect(loggerMock.error).toHaveBeenCalledWith('error test');
   });
 
+  it(
+    'logger.deprecation is called for all deprecated methods on the ' +
+      '_satellite object',
+    function () {
+      var logger = require('../logger');
+      var deprecationSpy = spyOn(logger, 'deprecation');
```

Comment:
For better or worse, this is different than how it's been done in the other tests (they use `injectHydrateSatelliteObject`). If you recognize the inconsistency and still want to move forward with it, I won't get in your way.

Reply:
FWIW, if I can get my intended coverage I’m totally open to switching back. Can we chat next week?

Reply:
see https://github.com/adobe/reactor-turbine/pull/121/commits/6cec604d5c2ca1b81de96b155d323df5e01c1e91